hack/build-antrea-linux-all.sh

```diff
@@ -53,6 +54,10 @@ case $key in
     PULL=true
     shift
     ;;
+    --testbed-type)
+    TESTBED_TYPE="$2"
+    shift 2
+    ;;
```

Comment:
we are not adding this flag to this script. This script is for building only, and we should not introduce anything which relates to a specific CI testbed.